source/fundamentals/crud/sort.txt

```diff
@@ -0,0 +1,109 @@
+        new Document().append("_id", 1)
+            .append("title", "The Brothers Karamazov").append("length", 824)
+            .append("author", "Dostoyevsky"),
+        new Document().append("_id", 2)
+            .append("title", "Les Misérables").append("length", 1462).append("author", "Hugo"),
+        new Document().append("_id", 3)
+            .append("title", "Atlas Shrugged").append("length", 1088).append("author", "Rand"),
+        new Document().append("_id", 4)
+            .append("title", "Infinite Jest").append("length", 1104).append("author", "Wallace"),
+        new Document().append("_id", 5)
+            .append("title", "Cryptonomicon").append("length", 918).append("author", "Stephenson"),
+        new Document().append("_id", 6)
+            .append("title", "A Dance with Dragons").append("length", 1104)
+            .append("author", "Martin")
+    ));
+
+Call the ``sort()`` function and specify the order with ``ascending()`` to ensure that the
+operation returns books in alphabetical order by title:
+
+.. code-block:: java
+  :emphasize-lines: 5
+
+    import com.mongodb.client.model.Sorts;
+
+    // define a cursor that can iterate through the results of the query
+    MongoCursor<Document> cursor = collection.find()
+        .sort(ascending("title")).iterator();
+    // print out items in alphabetical order
+    try {
+      while (cursor.hasNext()) {
+        System.out.println(cursor.next());
+      }
+    // close the cursor
+    } finally {
+      cursor.close();
+    }
+
+This sample code prints the documents in ascending order on the ``title`` field as follows:
+
+.. code-block:: java
+
+    Document{{_id=6, title=A Dance with Dragons, length=1104, author=Martin}}
+    Document{{_id=3, title=Atlas Shrugged, length=1088, author=Rand}}
+    Document{{_id=5, title=Cryptonomicon, length=918, author=Stephenson}}
+    Document{{_id=4, title=Infinite Jest, length=1104, author=Wallace}}
+    Document{{_id=2, title=Les Misérables, length=1462, author=Hugo}}
+    Document{{_id=1, title=The Brothers Karamazov, length=824, author=Dostoyevsky}}
+
+Sometimes, the order of two or more documents is ambiguous using a
+specified sort. Consider the case of sorting by length: in the above case, 
+both "A Dance with Dragons" and "Infinite Jest" have ``1104`` pages, so 
+the order in which ``sort()`` returns them is not guaranteed. To resolve ties in 
+your sorted results in a repeatable way, add additional fields to the 
+``Sorts.ascending()`` function call. 
+Call the ``sort()`` method, specifying ``length`` as 
+the first field by which to sort and the ``title`` field as the second field. In the event 
+that the ``length`` field comparison results in a tie, the operation compares the tied results 
+using the ``title`` field:
+
+
+.. code-block:: java
+  :emphasize-lines: 3
+    
+    // sort first by length, then title
+    MongoCursor<Document> cursor = collection.find()
+        .sort(ascending("length", "title")).iterator();
```

Comment:
You might want to show an example where one field is ascending and another is descending.